CMakeLists.txt

```diff
@@ -41,10 +53,19 @@ set(VCMI_VERSION_MAJOR 1)
 set(VCMI_VERSION_MINOR 0)
 set(VCMI_VERSION_PATCH 0)
 
+set(APP_SHORT_VERSION "${VCMI_VERSION_MAJOR}.${VCMI_VERSION_MINOR}")
+if(NOT VCMI_VERSION_PATCH EQUAL 0)
+	string(APPEND APP_SHORT_VERSION ".${VCMI_VERSION_PATCH}")
+endif()
+
 option(ENABLE_ERM "Enable compilation of ERM scripting module" ON)
 option(ENABLE_LUA "Enable compilation of LUA scripting module" ON)
 option(ENABLE_LAUNCHER "Enable compilation of launcher" ON)
-option(ENABLE_TEST "Enable compilation of unit tests" ON)
+if(APPLE_IOS)
+	set(BUNDLE_IDENTIFIER_PREFIX "" CACHE STRING "Bundle identifier prefix")
+else()
+	option(ENABLE_TEST "Enable compilation of unit tests" ON)
```

Comment:
No tests for iOS?

Reply:
Didn't bother checking them. They don't pass on desktop anyway.

If necessary, can be enabled later.